db/migrations/20241026183640_support_new_scanner.go

```diff
@@ -205,7 +205,7 @@ alter table media_file
 alter table media_file
 	add column tags jsonb default '{}' not null;
 alter table media_file
-	add column participant_ids jsonb default '{}' not null;
+	add column participations jsonb default '{}' not null;
```

Comment:
Just grammatically - is this actually participants (i.e. people participating in the performance), or is it situations in which the file is participating in something else (like a playlist)? Participations would mean the latter here.

Reply:
It was supposed to mean the former: People that has any kind of participation in a track/album, similar to contributions as opposed to contributors. I can change this, but can also take a while, as there's lots of references in the code :P